code/modules/projectiles/guns/projectile/rifle.dm

```diff
@@ -250,6 +250,43 @@
 	update_held_icon()
 	return
 
+
+/obj/item/gun/projectile/automatic/rifle/mk43
+	name = "Mark 43 service rifle"
```

Comment:
No specific names in the name. Use a generic name here and put the actual name in the desc.